### PR TITLE
Consistency

### PR DESCRIPTION
**Pull request by kacpermarzol**

### CODE REVIEW

File: lib/models/autodecoders/base_nerf.py [hunk 1]
```diff
@@ -455,6 +484,37 @@ def inverse_code(self, decoder, cond_imgs, cond_rays_o, cond_rays_d, dt_gamma=0,
...
+
+                for i in range(poses.shape[0]):
+                    M = poses[i]
+                    M = torch.from_numpy(M)
+                    M = M @ torch.Tensor([[-1, 0, 0, 0],
+                                          [0, 1, 0, 0],
+                                          [0, 0, 1, 0],
+                                          [0, 0, 0, 1]]).to(M.device)
+                    M = torch.cat(
+                        [M[:3, :3], (M[:3, 3:]) / 0.5], dim=-1)
```
Comment (piotrmwojcik) [inline lib/models/autodecoders/base_nerf.py]: Shouldn't make much difference for NERF but the complete matrix form is:
```
M = torch.cat([M[:3, :3], (M[:3, 3:] / 0.5)], dim=-1)
M = torch.cat([M, M.new_tensor([[0.0, 0.0, 0.0, 1.0]])], dim=-2)
```

File: lib/models/autodecoders/base_nerf.py [hunk 2]
```diff
@@ -414,6 +439,7 @@ def inverse_code(self, decoder, cond_imgs, cond_rays_o, cond_rays_d, dt_gamma=0,
         with module_requires_grad(decoder, False):
             n_inverse_steps = cfg.get('n_inverse_steps', 1000)
```
Comment (piotrmwojcik) [inline lib/models/autodecoders/base_nerf.py:419]: This is probably enough, but let's double check if` code` doesn't require copying and detaching.

File: lib/models/autodecoders/base_nerf.py [hunk 3]
```diff
@@ -438,6 +443,14 @@ def inverse_code(self, decoder, cond_imgs, cond_rays_o, cond_rays_d, dt_gamma=0,
             if show_pbar:
                 pbar = mmcv.ProgressBar(n_inverse_steps)
 
+            poses = [pose_spherical(theta, phi, -1.3) for phi, theta in fibonacci_sphere(6)]
+            poses = np.stack(poses)
+
+            if self.scheduler is not None:
```
Comment (piotrmwojcik) [inline lib/models/autodecoders/base_nerf.py]: Let's give this scheduler a more informative name

File: lib/models/autodecoders/base_nerf.py [hunk 4]
```diff
@@ -414,6 +418,7 @@ def inverse_code(self, decoder, cond_imgs, cond_rays_o, cond_rays_d, dt_gamma=0,
         with module_requires_grad(decoder, False):
             n_inverse_steps = cfg.get('n_inverse_steps', 1000)
             n_inverse_rays = cfg.get('n_inverse_rays', 4096)
+            n_inverse_rays_multi = cfg.get('n_inverse_rays_multi', 4096)
```
Comment (piotrmwojcik) [inline lib/models/autodecoders/base_nerf.py]: Let's name this so that it is clear that those rays are used for consistency loss
